src/mame/misc/magic10.cpp

```diff
@@ -737,6 +825,62 @@ static INPUT_PORTS_START( sgsafari )
 	PORT_DIPSETTING(        0x0000, DEF_STR( On ) )
 INPUT_PORTS_END
 
+static INPUT_PORTS_START( spetrix )
+	PORT_START("IN0")
+	PORT_BIT( 0x0001, IP_ACTIVE_LOW, IPT_POKER_HOLD1 )   PORT_NAME("Right - Hold 1")
+	PORT_BIT( 0x0002, IP_ACTIVE_LOW, IPT_POKER_HOLD2 )   PORT_NAME("Left - Hold 2 / Low")
+	PORT_BIT( 0x0004, IP_ACTIVE_LOW, IPT_POKER_HOLD3 )   PORT_NAME("Up - Hold 3")
+	PORT_BIT( 0x0008, IP_ACTIVE_LOW, IPT_POKER_HOLD4 )   PORT_NAME("Down - Hold 4 / High")
+	PORT_BIT( 0x0010, IP_ACTIVE_LOW, IPT_POKER_HOLD5 )   PORT_NAME("Rotate Left - Hold 5")
+	PORT_BIT( 0x0020, IP_ACTIVE_LOW, IPT_START1 )        PORT_NAME("Select - Start / Collect")
+	PORT_BIT( 0x0040, IP_ACTIVE_LOW, IPT_GAMBLE_BET )    PORT_NAME("Rotate Right - Bet / Collect")
+	PORT_BIT( 0x0080, IP_ACTIVE_LOW, IPT_UNUSED )
+	PORT_BIT( 0x0100, IP_ACTIVE_LOW, IPT_COIN1 )         PORT_NAME("Coin 1")
+	PORT_BIT( 0x0200, IP_ACTIVE_LOW, IPT_COIN2 )         PORT_NAME("Coin 2")
```

Comment:
Please don’t unnecessarily override default input names – it breaks localisation.